### ISSUE TITLE
Dependencies on windows

### ISSUE THREAD

**Issue by MoonDragon-MD** [FlorianEagox/weeablind#34]
Hi, I wanted to try this program, unfortunately I can't get it to work because of dependencies on windows, I downloaded "weeablind.1.0.zip" , I tried to create an env with python portable 3.10.5 and install the recommended windows dependencies, unfortunately there are versions that conflict. I was wondering if it is possible to already have a portable version with all the dependencies so that you can run "weeablind.exe" without problems?

**Comment by MoonDragon-MD** [comment]
I wanted to warn you that I keep seeing the error in red at startup but then I can get it to work, so there is something in the dependency check that is not working

**Comment by Monk1mmmmm** [comment]
Can you provide more details? @MoonDragon-MD

**Comment by MoonDragon-MD** [comment]
@MonX94 I downloaded [python portable](https://sourceforge.net/projects/portable-python/files/) (3.10.5) because I don't have space in the main system disk, then I start it the Console (Console-Launcher.exe) which is actually like a venv
and then I type: start weeablind.exe
The list of installed dependencies is this:
`pip list
Package                      Version
---------------------------- -----------
absl-py                      2.1.0
aiohttp                      3.9.5
aiosignal                    1.3.1
alembic                      1.13.1
altgraph                     0.17.4
annotated-types              0.6.0
antlr4-python3-runtime       4.9.3
anyascii                     0.3.2
anyio                        3.7.1
asteroid-filterbanks         0.4.0
astunparse                   1.6.3
async-timeout                4.0.3
attrs                        23.2.0
audioread                    3.0.1
audiotsm                     0.1.2
Babel                        2.15.0
bangla                       0.0.2
blis                         0.7.11
bnnumerizer                  0.0.2
bnunicodenormalizer          0.1.6
Brotli                       1.1.0
cachetools                   5.3.3
catalogue                    2.0.10
certifi                      2024.2.2
cffi                         1.16.0
charset-normalizer           3.3.2
click                        7.1.2
cloudpathlib                 0.16.0
colorama                     0.4.6
colorlog                     6.8.2
comtypes                     1.4.2
confection                   0.1.4
contourpy                    1.2.1
coqpit                       0.0.17
coqui-tts-trainer            0.1.1
cycler                       0.12.1
cymem                        2.0.8
Cython                       3.0.10
dateparser                   1.1.8
decorator                    5.1.1
docopt                       0.6.2
einops                       0.8.0
encodec                      0.1.1
espeakng                     1.0.3
exceptiongroup               1.2.1
ffmpeg                       1.4
ffmpeg-python                0.2.0
filelock                     3.14.0
flatbuffers                  1.12
fonttools                    4.51.0
frozenlist                   1.4.1
fsspec                       2024.3.1
future                       1.0.0
g2pkk                        0.1.2
gast                         0.4.0
gitdb                        4.0.11
GitPython                    3.1.43
glob2                        0.7
google-auth                  2.35.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
greenlet                     3.0.3
grpcio                       1.63.0
gruut                        2.2.3
gruut-ipa                    0.13.0
gruut-lang-de                2.0.0
gruut-lang-en                2.0.0
gruut-lang-es                2.0.0
gruut-lang-fr                2.0.2
gruut-lang-it                2.0.0
gruut-lang-ru                2.0.0
h11                          0.12.0
h2                           4.1.0
h5py                         3.11.0
hangul-romanize              0.1.0
hpack                        4.0.0
httpcore                     0.13.7
httpx                        0.19.0
huggingface-hub              0.23.0
humanize                     4.9.0
hyperframe                   6.0.1
HyperPyYAML                  1.2.2
idna                         3.7
inflect                      7.2.1
iniconfig                    2.0.0
intel-openmp                 2021.4.0
jamo                         0.4.1
jieba                        0.42.1
Jinja2                       3.1.4
joblib                       1.4.2
jsonlines                    1.2.0
julius                       0.2.7
keras                        2.9.0
Keras-Preprocessing          1.1.2
kiwisolver                   1.4.5
langcodes                    3.4.0
language_data                1.2.0
lazy_loader                  0.4
libclang                     18.1.1
librosa                      0.10.2
lightning                    2.2.4
lightning-utilities          0.11.2
llvmlite                     0.42.0
Mako                         1.3.3
marisa-trie                  1.1.1
Markdown                     3.6
markdown-it-py               3.0.0
MarkupSafe                   2.1.5
matplotlib                   3.8.4
mdurl                        0.1.2
mkl                          2021.4.0
more-itertools               10.2.0
mpmath                       1.3.0
msgpack                      1.0.8
multidict                    6.0.5
murmurhash                   1.0.10
mutagen                      1.47.0
networkx                     2.8.8
nltk                         3.8.1
norbert                      0.2.1
nose                         1.3.7
nostril                      0.1.1
num2words                    0.5.13
numba                        0.59.1
numpy                        1.26.4
oauthlib                     3.2.2
omegaconf                    2.3.0
opt-einsum                   3.3.0
optuna                       3.6.1
packaging                    24.0
pandas                       1.5.3
pefile                       2023.2.7
pillow                       10.3.0
pip                          24.3.1
plac                         1.4.3
platformdirs                 4.2.1
pluggy                       1.5.0
pooch                        1.8.1
preshed                      3.0.9
pretty-errors                1.2.25
primePy                      1.3
propcache                    0.2.0
protobuf                     3.9.2
psutil                       5.9.8
pyannote.audio               3.2.0
pyannote.core                5.0.0
pyannote.database            5.0.1
pyannote.metrics             3.2.1
pyannote.pipeline            3.0.1
pyasn1                       0.6.0
pyasn1_modules               0.4.0
PyAudio                      0.2.14
pycparser                    2.22
pycryptodomex                3.20.0
pydantic                     2.7.1
pydantic_core                2.18.2
pydub                        0.25.1
Pygments                     2.18.0
pyinstaller                  6.11.0
pyinstaller-hooks-contrib    2024.9
pynndescent                  0.5.12
pyparsing                    3.1.2
pypinyin                     0.51.0
pypiwin32                    223
pysbd                        0.3.4
pytest                       8.2.0
python-crfsuite              0.9.10
python-dateutil              2.9.0.post0
pytorch-lightning            2.2.4
pytorch-metric-learning      2.5.0
pyttsx3                      2.90
pytz                         2024.1
pywin32                      306
pywin32-ctypes               0.2.3
PyYAML                       6.0.1
regex                        2024.5.10
requests                     2.31.0
requests-oauthlib            2.0.0
rfc3986                      1.5.0
rich                         13.7.1
rsa                          4.9
ruamel.yaml                  0.18.6
ruamel.yaml.clib             0.2.8
safetensors                  0.4.3
scikit-learn                 1.4.2
scipy                        1.13.0
semver                       3.0.2
sentencepiece                0.2.0
setuptools                   58.1.0
shellingham                  1.5.4
six                          1.16.0
smart-open                   6.4.0
smmap                        5.0.1
sniffio                      1.3.1
sortedcontainers             2.4.0
soundfile                    0.12.1
soxr                         0.3.7
spacy                        3.7.4
spacy-legacy                 3.0.12
spacy-loggers                1.0.5
speechbrain                  1.0.0
spleeter                     2.4.0
SQLAlchemy                   2.0.30
srsly                        2.4.8
srt                          3.5.3
static-ffmpeg                2.7
SudachiDict-core             20240409
SudachiPy                    0.6.8
sympy                        1.13.1
tabulate                     0.9.0
tbb                          2021.12.0
tensorboard                  2.9.0
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorboardX                 2.6.2.2
tensorflow                   2.9.3
tensorflow-estimator         2.9.0
tensorflow-io-gcs-filesystem 0.31.0
termcolor                    2.5.0
thinc                        8.2.5
threadpoolctl                3.5.0
tokenizers                   0.19.1
tomli                        2.0.2
torch                        2.3.0
torch-audiomentations        0.11.1
torch-pitch-shift            1.2.4
torchaudio                   2.3.0
torchmetrics                 1.4.0
tqdm                         4.66.4
transformers                 4.40.2
typeguard                    4.2.1
typer                        0.3.2
typing_extensions            4.11.0
tzdata                       2024.1
tzlocal                      5.2
umap-learn                   0.5.6
urllib3                      2.2.1
wasabi                       1.1.2
weasel                       0.3.4
websockets                   12.0
Werkzeug                     3.0.3
wheel                        0.44.0
wrapt                        1.16.0
wxPython                     4.2.1
yarl                         1.9.4
yt-dlp                       2024.4.9`

Maybe ignore and try to use the system ones, do you have any idea in case how I can do that?

For example this program “[OneRingTranslator-installer](https://github.com/janvarev/OneRingTranslator-installer)” uses an interesting method to create a venv with the correct dependencies.

Thanks!
